specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/netapp.json

```diff
@@ -5473,5132 +7496,4967 @@
         },
-        "remoteVolumeResourceId": {
-          "type": "string",
-          "description": "The resource ID of the remote volume."
+        "type": {
+          "$ref": "#/definitions/CheckQuotaNameResourceTypes",
+          "description": "Resource type used for verification."
         },
-        "remoteVolumeRegion": {
+        "resourceGroup": {
           "type": "string",
-          "description": "The remote region for the other end of the Volume Replication."
+          "description": "Resource group name."
         }
-      }
+      },
+      "required": [
+        "name",
+        "type",
+        "resourceGroup"
+      ]
     },
-    "volumeRelocationProperties": {
-      "description": "Volume relocation properties",
+    "ReestablishReplicationRequest": {
       "type": "object",
+      "description": "Re-establish request object supplied in the body of the operation.",
       "properties": {
-        "relocationRequested": {
-          "type": "boolean",
-          "description": "Has relocation been requested for this volume"
-        },
-        "readyToBeFinalized": {
-          "type": "boolean",
-          "readOnly": true,
-          "description": "Has relocation finished and is ready to be cleaned up"
+        "sourceVolumeId": {
+          "type": "string",
+          "description": "Resource id of the source volume for the replication"
         }
       }
     },
-    "volumeSnapshotProperties": {
-      "description": "Volume Snapshot Properties",
+    "RegionInfo": {
       "type": "object",
+      "description": "Provides region specific information.",
       "properties": {
-        "snapshotPolicyId": {
-          "type": "string",
-          "description": "Snapshot Policy ResourceId"
+        "storageToNetworkProximity": {
+          "$ref": "#/definitions/RegionStorageToNetworkProximity",
+          "description": "Provides storage to network proximity information in the region."
+        },
+        "availabilityZoneMappings": {
+          "type": "array",
+          "description": "Provides logical availability zone mappings for the subscription for a region.",
+          "items": {
+            "$ref": "#/definitions/RegionInfoAvailabilityZoneMappingsItem"
+          },
+          "x-ms-identifiers": [
+            "availabilityZone"
+          ]
         }
       }
     },
-    "replicationStatus": {
-      "description": "Replication status",
+    "RegionInfoAvailabilityZoneMappingsItem": {
       "type": "object",
       "properties": {
-        "healthy": {
-          "type": "boolean",
-          "description": "Replication health check"
-        },
-        "relationshipStatus": {
-          "type": "string",
-          "description": "Status of the mirror relationship",
-          "enum": [
-            "Idle",
-            "Transferring"
-          ],
-          "x-ms-enum": {
-            "name": "RelationshipStatus",
-            "modelAsString": true
-          }
-        },
```

Comment:
The other 2 `RelationshipStatus` has 4 states -- Idel, Transferring, Failed, Unknown

Is this correct?

Reply:
Got it. I will first put this one as a separate enum e.g. named `VolumeReplicationRelationshipStatus` (let me know if you have any suggestion on naming, or whether we want to just make all 3 different enum at this TypeSpec)